src/chart/Column.js

```diff
@@ -1,83 +1,93 @@
 "use strict";
 (function (root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["d3", "./XYAxis", "../api/I2DChart", "css!./Column"], factory);
+        define(["d3", "./XYAxis", "../api/INDChart", "css!./Column"], factory);
     } else {
-        root.chart_Column = factory(root.d3, root.chart_XYAxis, root.api_I2DChart);
+        root.chart_Column = factory(root.d3, root.chart_XYAxis, root.api_INDChart);
     }
-}(this, function (d3, XYAxis, I2DChart) {
+}(this, function (d3, XYAxis, INDChart) {
     function Column(target) {
         XYAxis.call(this);
-        I2DChart.call(this);
-
-        this._linearGap = 25;
+        INDChart.call(this);
     }
     Column.prototype = Object.create(XYAxis.prototype);
     Column.prototype._class += " chart_Column";
-    Column.prototype.implements(I2DChart.prototype);
+    Column.prototype.implements(INDChart.prototype);
 
     Column.prototype.publish("paletteID", "default", "set", "Palette ID", Column.prototype._palette.switch(),{tags:['Basic','Shared']});
 
+    Column.prototype.testData2 = function () {
```

Comment:
Again, mistake from the merge, this has been moved into the XYAxis now (I think!)